lib/modules/csv_user_exporter.rb

```diff
@@ -36,6 +37,6 @@ def self.format_user_row(user, default=@@default_na)
   def self.create_filepath
     folder = Rails.root.join("public", "csv_exports")
     FileUtils.mkdir_p(folder)
-    folder.join("csv_user_export_#{DateTime.now.to_i}.csv")
+    folder.join("csv_user_export_#{DateTime.now.to_s}_#{SecureRandom.urlsafe_base64(5)}.csv")
```

Comment:
Noticed this method is used twice in two different files, may be you could abstract this out and create a CSV parent module which these two can inherit from and benefit from any abstracted helper functions

Reply:
Thanks @spencerldixon I will make that improvement!